dashboard/src/views/bench/NewBench.vue

```diff
@@ -20,14 +20,14 @@
 						Name your bench based on its purpose. For e.g., Personal Websites,
 						Staging Bench, etc.
 					</p>
-					<Input class="mt-4" type="text" v-model="benchTitle" />
+					<Input class="" type="text" v-model="benchTitle" />
```

Comment:
```suggestion
					<Input type="text" v-model="benchTitle" />
```
we don't need class if it's empty